.rubocop_todo.yml

```diff
@@ -1,17 +1,17 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config --exclude-limit 9999`
-# on 2018-12-26 19:18:36 -0500 using RuboCop version 0.61.1.
+# on 2020-01-14 23:50:04 -0500 using RuboCop version 0.76.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
```

Comment:
It was my understanding that this file shouldn't have any green in it, until we clear it out